### ISSUE TITLE
too many verts

### ISSUE THREAD

**Issue by mikyoupon** [Metastruct/outfitter#7]
There's a few model addons on workshop that, when selected, give off errors such as these

```
[Outfitter] CheckVVD	ValidateVVDVerts	models/captainbigbutt/vocaloid/rin	too many verts
[Outfitter] CheckVVD	ValidateVVDVerts	models/player/dewobedil/kancolle/zuikaku/default_p	too many verts
[Outfitter] CheckVVD	ValidateVVDVerts	models/player/dewobedil/junko_enoshima/default_p	too many verts
[Outfitter] CheckVVD	ValidateVVDVerts	models/player/shi/rin	too many verts
```

and prevent the player from selecting the model. Is there a planned bugfix for this?

**Comment by Noiwex** [comment]
Yes, it's possible to raise the limits.

**Comment by Python1320** [comment]
The too many verts is preventing major mass crashes. 
Basically if you wore those models and someone tossed paint buckets (or caused othe decals) at you everyone who currently sees you would crash.

**Comment by Noiwex** [comment]
r_drawmodeldecals 0

fixed this for you

**Comment by Noiwex** [comment]
now "too many verts" limit should be removed

**Comment by Noiwex** [comment]
Fixed in ddc518e.
